global.json

```diff
@@ -21,7 +21,7 @@
     "rollForward": "latestPatch"
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Arcade.Sdk": "8.0.0-beta.24075.5",
+    "Microsoft.DotNet.Arcade.Sdk": "8.0.0-beta.24081.5"
```

Comment:
```suggestion
    "Microsoft.DotNet.Arcade.Sdk": "8.0.0-beta.24081.5",
```

Reply:
Whoops! Thanks, didn't notice I stuffed up the merge :)